srcpkgs/xprop/template

```diff
@@ -9,8 +9,8 @@ short_desc="Property displayer for X"
 maintainer="Frank Steinborn <steinex@nognu.de>"
 license="MIT"
 homepage="https://xorg.freedesktop.org/wiki/"
-distfiles="${XORG_SITE}/app/${pkgname}-${version}.tar.bz2"
-checksum=9b92ed0316bf2486121d8bac88bd1878f16b43bd335f18009b1f941f1eca93a1
+distfiles="${XORG_SITE}/app/${pkgname}-${version}.tar.xz"
```

Comment:
homepage: https://gitlab.freedesktop.org/xorg/app/xprop

and replace $pkgname with xprop